services-custom/s3-transfer-manager/src/test/java/software/amazon/awssdk/transfer/s3/internal/DefaultS3CrtAsyncClientTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.&#10; &#10;Licensed under the Apache License, Version 2.0
+ * (the &quot;License&quot;).&#10;You may not use this file except in compliance with the License.&#10;A copy of the License is
+ *  located at&#10; &#10; http://aws.amazon.com/apache2.0&#10; &#10;or in the &quot;license&quot; file accompanying this file.
+ * This file is distributed&#10;on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either&#10;express
+ *  or implied. See the License for the specific language governing&#10;permissions and limitations under the License.
+ */
+
```

Comment:
nit: this looks very strange, some characters looks like they're XML escaped

Reply:
Ha, good catch. There is a weird issue with my Intellij copyright profile. Will fix